doc/designs/subordinate-ids.md

```diff
@@ -340,6 +352,16 @@ subordinate id entries for new users:
 $ ipa config-mod --user-default-subid=true
 ```
 
+Subordinate ID feature can be disabled completely. This is done with `ipa
+config-mod --addattr ipaconfigstring=SubID:Disable` command. After it is done,
+subordinate ID range can be removed with `ipa idrange-del` command and on the
+IPA server one have to run `ipa-server-upgrade` command to make sure internal
+DNA plugin configuration is removed as well. Finally, a new local ID range can
+be added to cover required part of the
```

Comment:
The sentence is missing its end.

Reply:
thanks, missed an update after rephrasing. Fixed now.